### PR TITLE
Update built-in-monitoring.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tune-performance.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ By default, SQL queries that take more than 300 milliseconds are considered as s
 
 To view slow queries in a cluster, perform the following steps:
 
-1. Navigate to the **SQL Diagnosis** page of a cluster.
+1. Navigate to the **Diagnosis** page of a cluster.
```
Comment (ran-huang) [inline tidb-cloud/tune-performance.md:26]: 设计图上还是 SQL Diagnosis 呢？

Comment (ran-huang) [inline tidb-cloud/tune-performance.md:26]:
```suggestion
1. Navigate to the **SQL Diagnosis** page of a cluster.
```

File: tidb-cloud/tune-performance.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ For more information, see [Slow Queries in TiDB Dashboard](https://docs.pingcap.
 
 To use the statement analysis, perform the following steps:
 
-1. Navigate to the **SQL Diagnosis** page of a cluster.
+1. Navigate to the **Diagnosis** page of a cluster.
```
Comment (ran-huang) [inline tidb-cloud/tune-performance.md:44]:
```suggestion
1. Navigate to the **SQL Diagnosis** page of a cluster.
```

File: tidb-cloud/tune-performance.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ For more information, see [Statement Execution Details in TiDB Dashboard](https:
 
 To view the key analytics, perform the following steps:
 
-1. Navigate to the **SQL Diagnosis** page of a cluster.
+1. Navigate to the **Diagnosis** page of a cluster.
```
Comment (ran-huang) [inline tidb-cloud/tune-performance.md:66]:
```suggestion
1. Navigate to the **SQL Diagnosis** page of a cluster.
```

File: tidb-cloud/tune-performance.md (hunk 1 above)
Comment (lilyjazz) [reply tidb-cloud/tune-performance.md]: 实际界面上是 Diagnosis

**Review by ran-huang — APPROVED** [review]

**Pull request merged**